csrc/scheduler/matmul_heuristic.h

```diff
@@ -25,42 +25,42 @@ class MatmulParams : public HeuristicParams {
   //!  parallelization will be done.
   enum class TileRasterizationOrder { RowMajor = 0, ColumnMajor = 1 };
 
-  //! A wrapper for double buffering config pieces
-  struct DoubleBufferOptions {
-    bool double_buffer_smem_write = false;
-    bool double_buffer_smem_read = false;
-    // This parameter controls the number of double buffering or circular
+  //! A wrapper for circular buffering config pieces
+  struct CircularBufferOptions {
+    bool circular_buffer_smem_write = false;
```

Comment:
Side note to myself: in another PR I believe we can remove this attribute since it should be equivalent to `smem_circular_buffer_stage > 1`.